baselines/getting_started/aws/aws_mods/demo.tfvars

```diff
@@ -0,0 +1,124 @@
+variable "mod_list" {
```

Comment:
This should be in variables.tf file, an example of the values should be in the demo.tfvars file. 

These default values can then be listed without comment. 

See:
- [example variables](https://github.com/turbot/tdk/blob/master/baselines/getting_started/aws/aws_check_regions/variables.tf)
- [example demo](https://github.com/turbot/tdk/blob/master/baselines/getting_started/aws/aws_check_regions/demo.tfvars)

Reply:
Updated the changes as per the examples.